site/_data/releases.yaml

```diff
@@ -1,7 +1,12 @@
 stable:
-  name: Darga-3
-  tag: darga-3
-  filename: darga-3
+  name: darga-4.1
+  tag: darga-4.1
+  filename: darga-4.1
+
+prerelease:
+  name: euwe-1-beta1.3
+  tag: euwe-1-beta1.3
+  finename: euwe-1-beta1.3
```

Comment:
Looks like I need to add a pre-release section to the download page, working on a PR for that now.